api/pkg/apis/v1alpha1/managers/configs/configs-manager_test.go

```diff
@@ -464,3 +480,97 @@ func TestObjectReference(t *testing.T) {
 	assert.NotNil(t, err2)
 	assert.Empty(t, val2)
 }
+
+func TestCircular(t *testing.T) {
```

Comment:
Test function names should be descriptive, such as TestCircularCatalogReferences